viewer/packages/rendering/src/glsl/base/isClipped.glsl

```diff
@@ -2,7 +2,7 @@
 uniform vec4 clippingPlanes[NUM_CLIPPING_PLANES];
 #endif
 
-bool isClipped(NodeAppearance nodeAppearance, vec3 point) {
+bool isClipped(vec3 point) {
 #if NUM_CLIPPING_PLANES > 0
   vec3 pointFlipped = -point;
   vec4 plane;
```

Comment:
The lines above sucks in my eyes! Why not simplify with early return and remove clipped? Cannot understand why this code has been approved.

Reply:
Hehe, not sure why.. It could be that they were afraid that shader threads running in parallel would diverge (one returns early, another one doesn't), which could cause some unfortunate performance loss, but not sure if that's a real concern or not :sweat_smile: